### PR TITLE
[USH-1699] Identify unlocalized strings in the web client and add them to en.json

### PR DESCRIPTION
**Pull request by MMwandigha**

### CODE REVIEW

File: apps/web-mzima-client/src/app/activity/bar-chart/bar-chart.component.ts [hunk 1]
```diff
@@ -19,18 +20,24 @@ export class BarChartComponent implements OnInit {
...
   public ngOnInit() {
     this.getPostStatistics(this.selectedFilter);
   }
 
+  private initializeFilters() {
+    this.filters = [
+      { displayNane: this.translate.instant('activity.categories'), value: 'tags' },
+      { displayNane: this.translate.instant('activity.survey'), value: 'form' },
+      { displayNane: this.translate.instant('activity.status'), value: 'status' },
+    ];
```
Comment (Angamanga) [inline apps/web-mzima-client/src/app/activity/bar-chart/bar-chart.component.ts:38]: Is it actually called displayNane? :D I see it was called that before too, maybe we should change it to displayName, or am I missing something important?

**Review by Angamanga — APPROVED** [review]
Well done! Approved!

File: apps/web-mzima-client/src/app/activity/bar-chart/bar-chart.component.ts (hunk 1 above)
Comment (MMwandigha) [reply apps/web-mzima-client/src/app/activity/bar-chart/bar-chart.component.ts:38]: Ah yes, I should change that. I almost thought it was named like that on purpose but I can still change it :)

Comment (Angamanga) [reply apps/web-mzima-client/src/app/activity/bar-chart/bar-chart.component.ts:38]: Actually, when thinking about it, its for the activity-view that is most likely going to change very soon, so lets leave it as it. Its out of scope for this ticket :)

Comment (MMwandigha) [reply apps/web-mzima-client/src/app/activity/bar-chart/bar-chart.component.ts:38]: Aah ok, actually I have already made the changes, but I think we can leave it at that

**Pull request merged**